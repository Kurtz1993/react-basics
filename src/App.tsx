interface AppProps {
  saludo: string;
  subtitulo?: string;
}

export default function App({ saludo, subtitulo = 'Soy un subt√≠tulo' }: AppProps) {
  return (
    <>
      <h1>{saludo}</h1>;<p>{subtitulo}</p>
    </>
  );
}
