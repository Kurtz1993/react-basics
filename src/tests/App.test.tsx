import { shallow } from 'enzyme';

import App from '../App';

describe('App', () => {
  test('should show the given greeting', () => {
    const greeting = 'Hola, soy Goku!';
    const wrapper = shallow(<App saludo={greeting} />);

    expect(wrapper).toMatchSnapshot();
  });
});
