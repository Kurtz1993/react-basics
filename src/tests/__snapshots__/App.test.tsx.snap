// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should show the given greeting 1`] = `
<Fragment>
  <h1>
    Hola, soy Goku!
  </h1>
  ;
  <p>
    Soy un subt√≠tulo
  </p>
</Fragment>
`;
